src/aleph/vm/orchestrator/run.py

```diff
@@ -57,31 +57,12 @@ async def create_vm_execution(vm_hash: ItemHash, pool: VmPool, persistent: bool
 
     logger.debug(f"Message: {message.json(indent=4, sort_keys=True, exclude_none=True)}")
 
-    try:
-        execution = await pool.create_a_vm(
-            vm_hash=vm_hash,
-            message=message.content,
-            original=original_message.content,
-            persistent=persistent,
-        )
-    except ResourceDownloadError as error:
-        logger.exception(error)
-        pool.forget_vm(vm_hash=vm_hash)
-        raise HTTPBadRequest(reason="Code, runtime or data not available") from error
-    except FileTooLargeError as error:
-        raise HTTPInternalServerError(reason=error.args[0]) from error
-    except VmSetupError as error:
-        logger.exception(error)
-        pool.forget_vm(vm_hash=vm_hash)
-        raise HTTPInternalServerError(reason="Error during vm initialisation") from error
-    except MicroVMFailedInitError as error:
-        logger.exception(error)
-        pool.forget_vm(vm_hash=vm_hash)
-        raise HTTPInternalServerError(reason="Error during runtime initialisation") from error
-    except HostNotFoundError as error:
-        logger.exception(error)
-        pool.forget_vm(vm_hash=vm_hash)
-        raise HTTPInternalServerError(reason="Host did not respond to ping") from error
```

Comment:
This method is also used on the method `start_persistent_vm()` and there aren't any other try catch handler there, so these lines are needed I think.
Maybe we can leave it here and remove it in the other `create_vm_execution_or_raise_http_error()` method, that does exactly the same controls.

Reply:
There is no catch handler in `start_persistent_vm()` but they are on the function that call it. The only place I found that used `start_persistent_vm()` and didn't have a try_catch was in `operate_reboot` so I switched it to use `create_vm_execution_or_raise_http_error()` (there is another place but it is in the debug cli so it make no sense to raise an HttpError) . in the end it is the same thing but the other way around (which make more senses with the name)

Reply:
I know this isn't related to confidential but it was masking an error while I was debugging